### ISSUE TITLE
Model definitions in separate files

### ISSUE THREAD

**Issue by victorgrey** [neumino/thinky#295]
Is there any way to spilt up model definitions into separate files and still define relations between them?

Given 
//in models/user.js
var User = thinky.createModel("User", {...});
module.exports = {User: User};

and
//in models/account.js
var Account = thinky.createModel("Account", {...});
module.exports = {Account: Account};

If I try to put this into user.js for example
var Account = require('./account.js').Account;
User.hasOne(Account, "account", "id", "account_id");

I get this error:
First argument of `hasOne` must be a Model

If both model definitions are in one file, it works fine.

**Comment by cadrogui** [comment]
According to docs, here is the way.

https://thinky.io/documentation/architecture/

regards.

**Comment by victorgrey** [comment]
Thanks - I had looked at those docs, but after your comment I went back and read them carefully again. I noticed that (1) the module.exports command exports the model directly (i.e. module.exports = User) whereas I was exporting an object (module.exports = { User: User }, and (2) that the call to require the foreign model and the Model.relation call came -after- the module.exports line. 

Fixing either one of these things didn't fix the problem, but fixing -both- did the trick. I really don't understand why, but maybe this will help someone else.
